korangar/src/graphics/passes/point_shadow/shader/entity_bindless.wgsl

```diff
@@ -64,12 +73,20 @@ fn vs_main(
     output.original_depth_offset = instance.depth_offset;
     output.original_curvature = instance.curvature;
     output.texture_index = instance.texture_index;
+    output.angle = instance.angle;
     return output;
 }
 
 @fragment
 fn fs_main(input: VertexOutput) -> @builtin(frag_depth) f32 {
-    let diffuse_color = textureSample(textures[input.texture_index], nearest_sampler, input.texture_coordinates);
+    var new_input = input.texture_coordinates;
```

Comment:
You need to document here, what and why you are doing here.

Reply:
I documented the two part that I changed